server/graphql/v1/mutations/expenses.js

```diff
@@ -478,11 +499,22 @@ export async function payExpense(remoteUser, args) {
     );
   }
 
-  const fxrate = await getFxRate(expense.collective.currency, host.currency);
   const feesInHostCurrency = {};
+  const fxrate = await getFxRate(expense.collective.currency, host.currency);
   const payoutMethod = await expense.getPayoutMethod();
   const payoutMethodType = payoutMethod ? payoutMethod.type : expense.getPayoutMethodTypeFromLegacy();
-  if (payoutMethodType === PayoutMethodTypes.PAYPAL && !args.forceManual) {
+
+  if (payoutMethodType === PayoutMethodTypes.BANK_ACCOUNT) {
+    const [connectedAccount] = await host.getConnectedAccounts({
+      where: { service: 'transferwise', deletedAt: null },
+    });
+    if (!connectedAccount) {
+      throw new Error('Host is not connected to Transferwise');
+    }
+    const quote = await paymentProviders.transferwise.getTemporaryQuote(connectedAccount, payoutMethod, expense);
+    // Notice this is the FX rate between Host and Collective, that's why we use `fxrate`.
+    fees.paymentProcessorFeeInCollectiveCurrency = Math.round((quote.fee / fxrate) * 100);
```

Comment:
Not a requested change: I've introduced a `floatAmountToCents` helper in https://github.com/opencollective/opencollective-api/blob/10b10bd8f12071bf66b6da184ebbab2311b5dc48/server/lib/math.js to do that. Someone not aware of the floating numbers problem may wonder why we `Math.round` and I think this little helper can help to clarify that.

Anyway, kudos for thinking about that :+1: